Core/clim-basic/extended-streams/text-formatting.lisp

```diff
@@ -31,23 +31,25 @@
 
 (defclass standard-page-layout ()
   ((%page-region :reader stream-page-region :writer (setf %page-region))
-   (margins :initform '(:left   (:absolute 0)
-                        :top    (:absolute 0)
+   (margins :initform '(:left   (:relative 0)
+                        :top    (:relative 0)
```

Comment:
Why is this changed? This does not seem purely syntactic.

Reply:
left/top is always located at 0/0, I've changed it to `:relative` to match bottom/right defaults (so the person who reads the code doesn't have to wonder why they are different).